### PR TITLE
[Back-port] Overcomming lvm specific problems with older python2

### PR DESCRIPTION
**Pull request by manvanthar**
Overcomming lvm specific probles with older python2 which cannot be enhanced

Signed-off-by: Manvanthara B Puttashankar <manvanth@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by manvanthar** [comment]
[root@ltczz401-lp4 disk]# avocado run lvsetup.py -m lvsetup.py.data/lvsetup.yaml 
JOB ID     : a487d6f974ad15f545133bb6f7baf6aeda8950cb
JOB LOG    : /root/tests/results/job-2019-10-06T10.14-a487d6f/job.log
 (1/6) lvsetup.py:Lvsetup.test;ext3-06d5: PASS (14.40 s)
 (2/6) lvsetup.py:Lvsetup.test;ext4-1ae5: PASS (5.31 s)
 (3/6) lvsetup.py:Lvsetup.test;xfs-ed84: PASS (5.93 s)
 (4/6) lvsetup.py:Lvsetup.test_lv_snapshot;ext3-06d5: PASS (16.06 s)
 (5/6) lvsetup.py:Lvsetup.test_lv_snapshot;ext4-1ae5: PASS (8.95 s)
 (6/6) lvsetup.py:Lvsetup.test_lv_snapshot;xfs-ed84: PASS (9.32 s)
RESULTS    : PASS 6 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 60.23 s
JOB HTML   : /root/tests/results/job-2019-10-06T10.14-a487d6f/results.html

File: io/disk/lvsetup.py [hunk 1]
```diff
@@ -128,14 +128,14 @@ def test(self):
...
 
-    @avocado.fail_on(lv_utils.LVException)
-    def test_vg_recreate(self):
-        """
-        Deactivate, export, import and activate a volume group.
-        """
-        self.create_lv()
-        lv_utils.vg_reactivate(self.vg_name, export=True)
-        self.mount_unmount_lv()
+#    @avocado.fail_on(lv_utils.LVException)
```
Comment (narasimhan-v) [inline io/disk/lvsetup.py]: @manvanthar instead of commenting out, this can be removed.

Comment (narasimhan-v) [reply io/disk/lvsetup.py]: @manvanthar I mean the whole function can be removed.

**Comment by manvanthar** [comment]
Changes in PR #1326

**Pull request closed without merging**